iceoryx_examples/singleprocess/single_process.cpp

```diff
@@ -1,4 +1,5 @@
-// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+// Copyright (c) 2020 - 2021 by Apex.AI Inc. All rights reserved.
+// Copyright (c) 2020 by Robert Bosch GmbH, Apex.AI Inc. All rights reserved.
```

Comment:
This might come from the merge of master. Please keep the original copyright holder first and remove the Apex.AI copyright from that line